updater/src/Updater/Generate/Template.purs

```diff
@@ -31,6 +31,7 @@ type Variables =
   , displayName :: String
   , displayTitle :: String
   , maintainers :: NonEmptyList String
+  , repo :: String -- not used
```

Comment:
We could use this, but change the parser so it has a default (`purescript-<package-name>`). Then we'd stop hardcoding that everywhere in the templates.

Reply:
Can I take care of that in a separate PR? I'm not exactly sure where I should and shouldn't be doing that in the template files.

Reply:
Yes, that's fine. Maybe we can make an issue or we could hop on a quick call and I can point them out.